health-services/PassiveGoals/app/src/main/java/com/example/passivegoals/PassiveGoalService.kt

```diff
@@ -0,0 +1,64 @@
+package com.example.passivegoals
+
+import android.util.Log
+import androidx.health.services.client.PassiveListenerService
+import androidx.health.services.client.data.*
```

Comment:
Nit: Avoid wildcard imports (don't know the full reasons why, do you?). Can set AS up to do this,

Reply:
Fixed.